sku.go

```diff
@@ -572,3 +572,13 @@ func (s *SKU) Equal(other *SKU) bool {
 		localErr != nil &&
 		otherErr != nil
 }
+
+// Includes returns true if the SKU's name is in the list of SKUs.
+func (s *SKU) Includes(skuList []SKU) bool {
+	for _, sku := range skuList {
+		if s.GetName() == sku.GetName() {
```

Comment:
EqualFold might be safer here, depending on where the list is coming from?

Reply:
The list would be populated thusly:

https://github.com/Azure/skewer/pull/25/files#diff-3cdaf65cd294be89999f768c30e5dedd31813ba12b2f5f05c82cf7ba2eaf90a4R72-R78

And then moved over to the karpenter provider repo for manual curation. I think Name is sufficient if we're going to want to have an easily manageable data structure to maintain.